include/itkVkComplexToComplex1DFFTImageFilter.hxx

```diff
@@ -112,7 +113,9 @@ void
 VkComplexToComplex1DFFTImageFilter<TInputImage, TOutputImage>::PrintSelf(std::ostream & os, Indent indent) const
 {
   Superclass::PrintSelf(os, indent);
-  os << indent << "DeviceID: " << m_DeviceID << std::endl;
+  os << indent << "UseVkGlobalConfiguration: " << m_UseVkGlobalConfiguration << std::endl;
+  os << indent << "Local DeviceID: " << m_DeviceID << std::endl;
+  os << indent << "Preferred DeviceID: " << this->GetDeviceID() << std::endl;
```

Comment:
If we are going to show the computed value `this->GetDeviceID()`, do we also want to show `VkGlobalConfiguration::GetDeviceID()` so that it is apparent even when `m_UseVkGlobalConfiguration` is `false`?  (I don't know the answer.) ... and similarly in the other places where this occurs.

Reply:
I went back and forth on this and decided to add it. It's a little strange to print out global values as part of the instance, but in this context it could be a significant help so that a user understands why a certain device is being used.